### PR TITLE
add modal to dynamic

### PR DESCRIPTION
**Pull request by mhpjay422**
Why this PR?
- Match design for embedded

This PR contains:
- Modal for embed

### CODE REVIEW

**Comment by mhpjay422** [comment]
https://github.com/FlatFilers/HCMShow/assets/12041898/17b1da56-b5c6-4f7e-83ad-c08eb146b8bc

File: components/sidebar-layout.tsx [hunk 1]
```diff
@@ -129,7 +129,7 @@ const SidebarLayout = ({ children }: Props) => {
   const workflowsNavigation = workflowItems(router);
 
   return (
-    <div className="h-screen w-screen bg-white">
+    <div className="h-screen w-screen bg-white overflow-hidden">
```
Comment (johnmosesman) [inline components/sidebar-layout.tsx:132]: What's this for? Does this affect the other views?

Comment (mhpjay422) [reply components/sidebar-layout.tsx:132]: The scrolling was happening at the highest tier parent, this div. I needed to disable it here and set it on dynamic so the modal height will be the same as the page height.

Edited to add overflow to pages that need it

Comment (johnmosesman) [reply components/sidebar-layout.tsx:132]: Visually that looks odd to me in your video, the header staying still vs the whole page scrolling. Playing with overflow is likely an anti pattern, I think there's something different you want here

**Comment by mhpjay422** [comment]
https://github.com/FlatFilers/HCMShow/assets/12041898/78cb61d0-d7e2-487b-91d2-bb492a78d6fd

**Comment by mhpjay422** [comment]
https://github.com/FlatFilers/HCMShow/assets/12041898/0ee67bcc-15f8-4d26-ba75-4d398f70b467

**Pull request merged**